R/FilterStateRange.R

```diff
@@ -532,6 +518,62 @@ RangeFilterState <- R6::R6Class( # nolint
       )
     },
 
+    server_inputs_fixed = function(id) {
+      moduleServer(
+        id = id,
+        function(input, output, session) {
+          logger::log_trace("RangeFilterState$server initializing, dataname: { private$dataname }")
+
+          finite_values <- reactive(Filter(is.finite, private$x_reactive()))
+          output$plot <- bindCache(
+            finite_values(),
+            cache = "session",
+            x = renderPlot(
+              bg = "transparent",
+              height = 25,
+              expr = {
+                private$unfiltered_histogram +
+                  if (!is.null(finite_values())) {
+                    ggplot2::geom_histogram(
+                      data = data.frame(x = Filter(is.finite, private$x_reactive())),
```

Comment:
Please fix also the same line in server_inputs
```suggestion
                      data = data.frame(x = finite_values()),
```